cmd/lakectl/cmd/config/config.go

```diff
@@ -0,0 +1,90 @@
+package config
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/spf13/viper"
+)
+
+type configuration struct {
```

Comment:
can you fill all the fields metadata so it will be aligned. it is not required, but it will prevent future break if somebody wil refactor the field name.

Reply:
Done.